luigi_pipeline/lib/model/seqr_mt_schema.py

```diff
@@ -109,31 +109,48 @@ def genotypes(self):
     def samples_no_call(self):
         return self._genotype_filter_samples(lambda g: g.num_alt == -1)
 
-    @row_annotation(fn_require=genotypes, multi_annotation=True)
+    @row_annotation(fn_require=genotypes)
     def samples_num_alt(self, start=1, end=3, step=1):
```

Comment:
This is an example of removing the multiple annotations option. With it, we can return 
```
{
  'samples_num_alt_1': [],
  'samples_num_alt_2': []
}
```

However, I found out that we can store it nested in the mt `num_alt: {1: [], 2: []}` and flatten it when we export it to ES. This would allow us to `remove multi_annotations`.